youtube-client/src/app/auth/pages/auth/auth.component.html

```diff
@@ -0,0 +1,23 @@
+<app-header></app-header>
+<div class="auth">
+  <div class="auth-wrapper">
+    <h3 class="auth-title">Login</h3>
+    <form [formGroup]="form" (ngSubmit)="addLogin()" class="auth-form">
```

Comment:
if user is already logged, you shouldn't show him login form, but only `logout` button